src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_sup.erl

```diff
@@ -11,9 +11,12 @@
 -export([start_link/0]).
 
 %% supervisor callbacks
--export([init/1]).
+-export([init/1,
+         default_resource_init/0]).
```

Comment:
Does this still need to be exported? I can't find reference to it outside of this file.